core/core_bind.cpp

```diff
@@ -30,6 +30,9 @@
 
 #include "core_bind.h"
 
+#include "core/object/object.h"
```

Comment:
```suggestion
```
Already included 

Are you using some tool in your IDE to automatically add includes? I'd suggest turning it off or checking that it actually makes the right suggestions, this is already included